it/src/test/java/com/linecorp/centraldogma/it/mirror/git/MirrorTestUtils.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.centraldogma.it.mirror.git;
+
+import java.io.IOException;
+
+import javax.annotation.Nullable;
+
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.lib.FileMode;
+import org.eclipse.jgit.lib.ObjectId;
+import org.eclipse.jgit.lib.ObjectReader;
+import org.eclipse.jgit.revwalk.RevWalk;
+import org.eclipse.jgit.treewalk.TreeWalk;
+
+public final class MirrorTestUtils {
+
+    @Nullable
+    public static byte[] getFileContent(Git git, ObjectId commitId, String fileName) throws IOException {
```

Comment:
Technically, this is not related to mirroring. How to rename the class name into `GitTestUtil` for clarification.